ci_scripts/base_util.sh

```diff
@@ -10,19 +10,20 @@
 set -e
 
 # Paths for the Helm chart resources
-CHART_FILE_PATH="$SCRIPT_DIR/../helm-charts/splunk-otel-collector/Chart.yaml"
-VALUES_FILE_PATH="$SCRIPT_DIR/../helm-charts/splunk-otel-collector/values.yaml"
+ROOT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/../"
+CHART_FILE_PATH="${ROOT_DIR}helm-charts/splunk-otel-collector/Chart.yaml"
+VALUES_FILE_PATH="${ROOT_DIR}helm-charts/splunk-otel-collector/values.yaml"
 
 # Set default OWNER to "signalfx" if not already set
 : "${OWNER:=signalfx}"  # Sets OWNER to "signalfx" if it is not already set
 
 # Debug mode is off by default but can be enabled with --debug
-: "${DEBUG_MODE:=0}"  # Sets DEBUG_MODE to 0 if it is not already set
+: "${DEBUG_MODE:=1}"  # Sets DEBUG_MODE to 0 if it is not already set
 
 # Iterate over all arguments of the calling script
 for arg in "$@"; do
     if [[ "$arg" == "--debug" ]]; then
-        DEBUG_MODE=1  # Enable debug mode
+        DEBUG_MODE=0  # Enable debug mode
```

Comment:
should that be reverted?

Reply:
I really want this change to cut down on lines in general. I'll revert it for now in this PR and add it in a future PR.

Reply:
Done